e2e_tests/create_namespace.py

```diff
@@ -13,6 +13,9 @@
 
 def test_cluster(cluster, oc_map, ns_under_test):
     oc = oc_map.get(cluster)
+    if not oc:
+        logging.log(level=oc.log_level, msg=oc.message)
+        return None
```

Comment:
So, this is a test... when we can't get an actionable `OC()`, doesn't that mean that the test failed?
At the end, this is just about the message we log, so no big deal.